app/src/css/style.css

```diff
@@ -1,43 +1,246 @@
 @import "@uswds/uswds/dist/css/uswds.css";
 
+header {
+  display: flex;
+  flex-direction: column;
+}
+
+main {
+  min-height: 70vh;
+}
+
+
+/* Header */
+.usa-header {
+  align-items: center;
+  display: flex;
+}
+
+.usa-navigation-container {
+  display: flex;
+  width: 90%
+}
+
+.usa-nav-bar {
+  width: 90%;
+}
+
 .title {
+  align-items: center;
   display: flex;
-  align-items: baseline;
 }
+
+h1 {
+  color: #093f8e;
+  font-size: 1.9em;
+  margin: 3% 2% 0 2%
+}
+
 .logo {
   width: 120px;
   height: 40px;
   margin-right: 10px;
 }
-.github-information {
+
+.usa-logo {
+  font-size: 1.5em;
+  margin: 1% 0 1% 0;
+}
+
+.usa-nav__primary-item {
   display: flex;
-  flex-direction: row;
-  justify-content: space-evenly;
+  justify-content: flex-end;
+  background-color: #fff;
+}
+
+.usa-nav {
+  opacity: 1;
+}
+
+.usa-nav.is-hidden {
+  opacity: 1;
 }
-.stat-container {
+
+.usa-icon {
+  fill: #000;
+}
+
+
+/* Home page main */
+.grid-container {
   display: flex;
   flex-direction: column;
   align-items: center;
 }
+
+.home-page-description {
+  margin: 3%;
+  width: 90%;
+}
+
+.description-styling {
+  display: flex;
+  justify-content: center;
+  font-size: 1.2em;
+}
+
+h2 {
+  margin-bottom: 0;
+}
+
+/* Cards */
+.usa-card-group-home {
+  display: flex;
+  flex-direction: row;
+  list-style-type:none;
+  margin-left: -80px;
+}
+
+.usa-card {
+  margin: 0 1% 0 1%;
+}
+
+.usa-card-outer {
+  display: flex;
+  background-color: #043884;
+  border-radius: 70px;
+  height: 11vw;
+  width: 25vw;
+  box-shadow: rgba(50, 50, 93, 0.25) 0px 2px 5px -1px, rgba(0, 0, 0, 0.3) 0px 1px 3px -1px;
+}
+
+.usa-card-middle {
+  display: flex;
+  background-color: #5C85C1;
+  border-radius: 70px;
+  height: 100%;
+  margin-left: -1px;
+  width: 97%;
+}
+
+.usa-card-middle:hover {
+  background-color: #E7C000;
+}
+
+.usa-card-outer:active {
+  box-shadow: none;
+}
+
+.href-home {
+  text-decoration:none;
+  width: 96%;
+}
+
+.href-home:focus {
+  outline:none;
+}
+
+
+.usa-card__container {
+  align-items: center;
+  background-color: #E0EAF8;
+  border-radius: 70px;
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+  margin: 0;
+  margin-left: -1px;
+  padding: 0;
+  width: 100%;
+}
+
+.home-icon-container {
+  align-items: center;
+  background-color: #fff;
+  border-radius: 50%;
+  display: flex;
+  height: 100px;
+  justify-content: center;
+  margin-left: 6%;
+  width: 100px;
+}
+
+.usa-card__header {
+  margin: 15% 1% 10% 5%;
+  padding: 0;
+}
+
+.usa-card__body {
+  padding: 0;
+}
+
+.usa-card-heading-home {
+  font-size: 1.2em;
+}
+
+/* Footer */
```

Comment:
Nitpicks below for the footer:
 - Change the color of the link when hovered, it's hard to make out
<img width="823" alt="Screenshot 2024-12-02 at 9 49 10 AM" src="https://github.com/user-attachments/assets/89071bfe-7f84-4d1f-815c-17a50dae2810">
- For links that have not been clicked on, change to a color with higher contrast
<img width="625" alt="Screenshot 2024-12-02 at 9 48 25 AM" src="https://github.com/user-attachments/assets/7394082b-8468-40a7-bbc8-469cd7a312f2">

Reply:
The link colors are resolved on the next PR.